README.md

```diff
@@ -0,0 +1,78 @@
+# smartd
+
+This role installs and configures `smartd`. 
+In the default configuration, all devices will be tested and observed, but no mails will be sent.
+
+## Requirements
+
+A Debian or Arch based distribution.
+`/usr/bin/mail` has to exist and be configured to be able to send report mails.
+
+## Role Variables
+
+This role only has one top-level var called `smartd_devices`, which is a dict of dicts to specify device monitoring.
```

Comment:
The role doesn't "have" the var, it expects/requires/... it

Reply:
Yes, it *uses* or *needs* the var, though it is already set through the defaults.